libs/utils/utils.js

```diff
@@ -1059,10 +1104,14 @@ async function checkForPageMods() {
   const { init } = await import('../features/personalization/personalization.js');
   await init({
     mepParam, mepHighlight, mepButton, pzn, promo, target,
-  });
+  }, targetInteractionPromise);
```

Comment:
Why add an extra param if you already have a nice object-param you can add the targetInteractionPromise to?